README.md

```diff
@@ -22,7 +22,7 @@ Restart Xming and start the container with the following command:
 
 ##### Linux
 To use graphics, make sure you are in an X11 session and run the following command: 
-`docker run -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix --rm -it --user $(id -u) rootproject/root`
+`docker run -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix --rm -it --user $(id -u) --entrypoint root rootproject/root`
```

Comment:
More common than using `--entrypoint` is to just pass the command after the image name:
```
docker run -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix --rm -it --user $(id -u) rootproject/root root
```